server/src/services/shared-link.service.ts

```diff
@@ -185,7 +185,7 @@ export class SharedLinkService {
 
     const sharedLink = await this.findOrFail(auth.sharedLink.userId, auth.sharedLink.id);
     const assetId = sharedLink.album?.albumThumbnailAssetId || sharedLink.assets[0]?.id;
-    const assetCount = sharedLink.assets.length ?? sharedLink.album?.assets.length ?? 0;
+    const assetCount = sharedLink.assets.length || sharedLink.album?.assets.length || 0;
```

Comment:
When share link is a whole album, `sharedLink.assets.length` is 0 and `sharedLink.album?.assets.length` is the real asset count. But `0 ?? 10 ?? 0 === 0`, so I changed to `0 || 10 || 0 === 10`.

Reply:
@danieldietzler see another reason why `||` is better than `??`. Just causes headaches most of the time imo.

Reply:
Haha! I think there are valid scenarios in which you specifically *don't* want `||` but yeah, by default I see your point